src/components/skills-quiz/tests/SearchProgramCard.test.jsx

```diff
@@ -140,28 +135,32 @@ const SearchProgramCardWithContext = ({
 
 describe('<SearchProgramCard />', () => {
   test('renders the correct data', async () => {
-    let containerDOM = {};
-    await act(async () => {
-      const { container } = renderWithRouter(
-        <SearchProgramCardWithContext
-          index={testIndex}
-        />,
-      );
-      containerDOM = container;
-    });
+    const { container, history } = renderWithRouter(
+      <SearchProgramCardWithContext
+        index={testIndex}
+      />,
+    );
+
+    const searchProgramCard = await screen.findByTestId('search-program-card');
+    expect(searchProgramCard).toBeInTheDocument();
 
     expect(screen.getByText(PROGRAM_TITLE)).toBeInTheDocument();
     expect(screen.getByAltText(PROGRAM_AUTHOR_ORG.name)).toBeInTheDocument();
+    expect(screen.getByText(PROGRAM_AUTHOR_ORG.name));
```

Comment:
[curious] similar to above

`expect(screen.getByText(PROGRAM_AUTHOR_ORG.name));`
vs
`expect(screen.getByText(PROGRAM_AUTHOR_ORG.name)).toBeInTheDocument();`

Reply:
Fixed. in upcoming commit.